sdv/multi_table/base.py

```diff
@@ -233,6 +194,14 @@ def _validate_table_name(self, table_name):
         if table_name not in self._table_synthesizers:
             raise InvalidDataError([f"Table '{table_name}' is not present in the metadata."])
 
+    def _get_all_foreign_keys(self, table_name):
```

Comment:
why do we need this method here? It's already in the metadata

Reply:
I'm just moving it down the file because it gets used by the method below, so it was misplaced up there.

Reply:
why doesn't the method below just use the metadata's version of the method?